index.html

```diff
@@ -1420,16 +1422,20 @@ <h3>Multi-Cloud</h3>
     </section>
 
     <section id="virtual-thing">
-      <h3>Virtual Things</h3>    
-      <section class="ednote" title="TODO">
-      <h3>TODO</h3>
-      Create a description and a diagram for a <a>virtual thing</a> scenario,
-      as described in the use case for 
-      <a href="https://w3c.github.io/wot-usecases/#USE-CASES/Retail-virtual-thing">Virtual Thing</a>
+      <h3>Virtual Things</h3>   
+	<p>
+	 A virtual thing is an abstraction of a physical entity that does not have a direct physical counterpart.
```

Comment:
could just say "a virtual thing is an abstraction that does not have a direct physical counterpart".  I had to scratch my head over the "abstraction of a physical entity that does not have a direct physical counterpart".  So does it have a physical referent, or not?  I think you're trying to say it does not, so...